jsonschema-module-microprofile-openapi-3/src/test/resources/com/github/victools/jsonschema/module/microprofile/openapi3/integration-test-result-Foo.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$defs": {
+        "referenceToPerson": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "the person's name"
+                }
+            },
+            "title": "reference title",
+            "description": "the foo's person",
+            "additionalProperties": false
```

Comment:
I had to define this `additionProperties` which is not defined on Swager 2.x side. I do not know why. Should it be defined ? 
https://github.com/victools/jsonschema-generator/blob/main/jsonschema-module-swagger-2/src/test/resources/com/github/victools/jsonschema/module/swagger2/integration-test-result-Foo.json#L13

Reply:
`"additionalProperties". false` is always dangerous, as it can easily lead to an invalid or too strict schema.
Would be good to understand where this is coming from and avoid it if possible.